Assets/Scripts/Asteroid/AsteroidMovement.cs

```diff
@@ -26,16 +27,31 @@ private void OnCollisionEnter2D(Collision2D col)
     {
         if (col.gameObject.CompareTag("ShipBullet"))
         {
+            int shipHealth = GameObject.Find("Ship").GetComponent<Ship>().shipHealth;
             asteroidHealth -= 1;
             if (asteroidHealth == 0)
             {
                 Destroy(gameObject);
+
                 if (Random.Range(0.00f, 1.00f) < 0.25)
```

Comment:
Let's turn 0.25 into a public variable that can be adjusted. Hardcoded values are bad. Please name appropriately.